ui/app/router.js

```diff
@@ -94,14 +94,16 @@ Router.map(function() {
           this.route('index', { path: '/' });
           this.route('configuration');
           // because globs / params can't be empty,
-          // we have to special-case ids of '' with thier own routes
+          // we have to special-case ids of '' with their own routes
           this.route('list-root', { path: '/list/' });
           this.route('create-root', { path: '/create/' });
           this.route('show-root', { path: '/show/' });
           this.route('edit-root', { path: '/edit/' });
 
           this.route('list', { path: '/list/*secret' });
           this.route('show', { path: '/show/*secret' });
+          this.route('metadata', { path: '/metadata/*secret' });
```

Comment:
two new routes. We thought a lot about the different configurations of routing and this was the easiest and most straight forward.